healthchecks/types.go

```diff
@@ -137,3 +150,34 @@ func (healthCheck HttpHealthCheck) Run(host Host) error {
 		return errors.New(fmt.Sprintf("Got non 2xx status code (%s)", resp.Status))
 	}
 }
+
+func (pre PreActivationCheck) GetDescription() string {
+	return pre.Description
+}
+
+func (pre PreActivationCheck) GetPeriod() int {
+	return pre.Period
+}
+
+func (pre PreActivationCheck) Run(host Host) error {
```

Comment:
Ideally we could merge this with `Run` from the old health checks, while still using these new error messages

Reply:
Maybe it's the calling function (`runCheckUntilSuccess`?) could prefix the health check errors with e.g. `Pre-activation check error`, or maybe it'll be too much of a hassle